pass-data-client/src/main/java/org/eclipse/pass/support/client/ModelUtil.java

```diff
@@ -31,14 +30,18 @@ public static DateTimeFormatter dateTimeFormatter() {
      * award number.
      * @param awardNumber award number to normalize
      * @return normalized award number
-     * @throws IOException if the award number cannot be normalized
      */
-    public static String grantAwardNumberNormalizer(String awardNumber) throws IOException {
-        if (StringUtils.isEmpty(awardNumber)) {
```

Comment:
I've started using the StringUtils for checks of null/empty as a recommendation from Russ. I think for this particular case it should be StringUtils.isBlank(). Since there could be multiple empty whitespaces. 

What you have works too. Everything else looks good!